doc/classes/@GlobalScope.xml

```diff
@@ -2550,6 +2550,155 @@
 		<constant name="JOY_AXIS_MAX" value="10" enum="JoyAxis">
 			The maximum number of game controller axes: OpenVR supports up to 5 Joysticks making a total of 10 axes.
 		</constant>
+		<constant name="JOY_MODEL_INVALID" value="-1" enum="JoyModel">
+			An invalid or unknown game controller model.
+		</constant>
+		<constant name="JOY_MODEL_XBOX_GENERIC" value="0" enum="JoyModel">
+			A standard game controller model with the button layout similar to Xbox controllers.
+			A game controller with this model ID has A/B/X/Y face buttons, a directional pad, and it might have Back/Guide/Start buttons, left/right shoulder buttons and triggers.
```

Comment:
```suggestion
			A game controller with this model ID has A/B/X/Y face buttons, a directional pad, and it might have Back/Guide/Start buttons, left/right shoulder buttons, and triggers.
```